packages/augur-tools/docker-builder/Dockerfile

```diff
@@ -25,11 +25,15 @@ COPY --from=augur-build /augur /augur
 COPY --from=augur-build /opt /opt
 COPY --from=augur-build /usr/local /usr/local
 COPY --from=core-builder /app/output augur/packages/augur-core/output
+COPY --from=core-builder /app augur/packages/augur-core
```

Comment:
Can't just take it from augur-build, because it wasn't fully built (since it intentionally skips interfaces generated from solidity and the like)